src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -222,6 +222,21 @@ private AnomalyDetectorSettings() {}
             Setting.Property.Dynamic
         );
 
+    // for a batch operation, we want all of the bounding box in-place for speed
+    public static final double BATCH_BOUNDING_BOX_CACHE_RATIO = 1;
+
+    // for a real-time operation, we trade off speed for memory as real time opearation
+    // only has to do one update/scoring per interval
+    public static final double REAL_TIME_BOUNDING_BOX_CACHE_RATIO = 0;
```

Comment:
Does this mean we need to rebuild bounding box for every realtime job run? How much more memory will cost if we open this bounding box cache per RCF model?

Reply:
Different config have different numbers. For 30 trees, 2 dimensions, using full bounding box costs 935 KB, while not using bounding box  costs 322 KB.

Reply:
Yes, it means we need to rebuild for every real time job run.